Tribler/Test/Core/Modules/MetadataStore/test_channel_download.py

```diff
@@ -59,6 +59,6 @@ def test_channel_update_and_download(self):
         with db_session:
             # There should be 4 torrents + 1 channel torrent
             channel2 = self.session.lm.mds.ChannelMetadata.get(public_key=database_blob(payload.public_key))
-            self.assertEqual(5, len(list(self.session.lm.mds.TorrentMetadata.select())))
-            self.assertEqual(1562257279010, channel2.timestamp)
+            self.assertEqual(8, self.session.lm.mds.ChannelNode.select().count())
```

Comment:
Why are there 8 `ChannelNode` objects in this test? This is not consistent with the comment on line 60.

Reply:
I've added some nested channels to the test .mdblobs and forgot to update the comment.